syntax = "proto3";

message NotifyClientRequest {
    string notification_message = 1;
}

message NotifyClientResponse {
    enum Status {
        SUCCESS = 0;
        FAILED = 1;
    }
    Status status = 1;
}

service MarketService {
    rpc NotifyClient(NotifyClientRequest) returns (NotifyClientResponse) {}
}
